x-pack/package.json

```diff
@@ -202,6 +202,7 @@
     "inline-style": "^2.0.0",
     "intl": "^1.2.5",
     "io-ts": "^1.4.2",
+    "jest-cli": "^24.1.0",
```

Comment:
__question/issue:__ why do we need `jest-cli` in non-dev dependencies?

Reply:
seems like a mistake when updated version via yarn cli. will fix now